Platform/ADLINK/ComHpcAltPkg/ComHpcAlt.dsc

```diff
@@ -0,0 +1,444 @@
+# and dbx, most of which can be downloaded directly:
+# https://github.com/pftf/RPi4/blob/master/.github/workflows/linux_edk2.yml#L50-L58
+#
+# Note that, because we sure don't want any third party (including
+# ourselves) to have control over somebody else's machine when it comes
+# to Secure Boot, we always generate a new PK as part of the build process and then discard the private key altogether.
+#
+# Then, at EDK2 build time, you just need to feed the
+# -D SECURE_BOOT_ENABLE=TRUE option along with something like
+# -D DEFAULT_KEYS=TRUE -D PK_DEFAULT_FILE=$WORKSPACE/keys/pk.cer
+# -D KEK_DEFAULT_FILE1=$WORKSPACE/keys/ms_kek.cer
+# -D DB_DEFAULT_FILE1=$WORKSPACE/keys/ms_db1.cer
+# -D DB_DEFAULT_FILE2=$WORKSPACE/keys/ms_db2.cer
+# -D DBX_DEFAULT_FILE1=$WORKSPACE/keys/arm64_dbx.bin:
+# https://github.com/pftf/RPi4/blob/master/.github/workflows/linux_edk2.yml#L64-L65
+#
+# And with this, you should have a UEFI firmware that both Windows and
+# Linux are happy with when it comes to Secure Boot.
+
+!include MdePkg/MdeLibs.dsc.inc
+
+# Include default Ampere Platform DSC file
+!include Silicon/Ampere/AmpereAltraPkg/AmpereAltraPkg.dsc.inc
+
+################################################################################
+#
+# Specific Platform Library
+#
+################################################################################
+[LibraryClasses]
+  #
+  # Capsule Update requirements
+  #
+  BmpSupportLib|MdeModulePkg/Library/BaseBmpSupportLib/BaseBmpSupportLib.inf
+  DisplayUpdateProgressLib|MdeModulePkg/Library/DisplayUpdateProgressLibGraphics/DisplayUpdateProgressLibGraphics.inf
+  CapsuleLib|MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleLib.inf
+  EdkiiSystemCapsuleLib|SignedCapsulePkg/Library/EdkiiSystemCapsuleLib/EdkiiSystemCapsuleLib.inf
+  FmpAuthenticationLib|SecurityPkg/Library/FmpAuthenticationLibPkcs7/FmpAuthenticationLibPkcs7.inf
+  IniParsingLib|SignedCapsulePkg/Library/IniParsingLib/IniParsingLib.inf
+  PlatformFlashAccessLib|Silicon/Ampere/AmpereAltraPkg/Library/PlatformFlashAccessLib/PlatformFlashAccessLib.inf
+  ShellCEntryLib|ShellPkg/Library/UefiShellCEntryLib/UefiShellCEntryLib.inf
+  OemMiscLib|Platform/ADLINK/ComHpcAltPkg/Library/OemMiscLib/OemMiscLib.inf
+
+  #
+  # ACPI Libraries
+  #
+  AcpiLib|EmbeddedPkg/Library/AcpiLib/AcpiLib.inf
+
+  OemMiscLib|Platform/ADLINK/ComHpcAltPkg/Library/OemMiscLib/OemMiscLib.inf
+
+  #
+  # EFI Redfish drivers
+  #
+!if $(NETWORK_ENABLE) == TRUE
+!if $(REDFISH_ENABLE) == TRUE
+  RedfishContentCodingLib|RedfishPkg/Library/RedfishContentCodingLibNull/RedfishContentCodingLibNull.inf
+  RedfishPlatformHostInterfaceLib|RedfishPkg/Library/PlatformHostInterfaceLibNull/PlatformHostInterfaceLibNull.inf
+!endif
+!endif
+
+  #
+  # Library for POST Code converting
+  #
+  ReportStatusCodeLib|MdeModulePkg/Library/DxeReportStatusCodeLib/DxeReportStatusCodeLib.inf
+
+  #
+  # Pcie Board
+  #
+  BoardPcieLib|Library/BoardPcieLib/BoardPcieLib.inf

```

Comment:
I think adding `$PWD/edk2-platforms/Platform/${MANUFACTURER}/${BOARD_NAME}Pkg` to `PACKAGES_PATH` makes difficult to read. I cannot identify which package the BoardPcieLib instance belong to immediately.

And, I think it may be hard to manage and easy to get conflicted when we add multiple packages into the `PACKAGES_PATH`

Reply:
I've updated the paths from `Library/...` to `Platform/ADLINK/ComHpcAltPkg/Library/...` so ${BOARD_NAME}Pkg no longer needs to be in $PACKAGES_PATH.